fairing/kubernetes/manager.py

```diff
@@ -113,6 +113,7 @@ def get_service_external_endpoint(self, name, namespace, selectors=None):
             logger.error("error getting status for {} {}".format(name, str(e)))
 
     def log(self, name, namespace, selectors=None, container='', follow=True):
+        tail = ''
```

Comment:
clean up unused vars?

Reply:
Thanks for the review @Jeffwan!

The line was added because that variable is referred to at line https://github.com/vjrantal/fairing/blob/azure-support/fairing/kubernetes/manager.py#L163 but assigned only within certain if-blocks. Without the added line, the error message I got was:

```
The local variable `tail` was referenced before assignment
```

The root cause for the error is the same as what one would see if executing this kind of Python code https://gist.github.com/vjrantal/17c57a1ff48b08a7a497b8061805996a.

If you have suggestions how to refactor the `log`-function in another way to get rid of this issue, I can amend my commit accordingly.